docs/usage/export-documents.md

```diff
@@ -5,3 +5,65 @@
     We are doing our best to complete this page. 
     If you want to participate, don't hesitate to join the [Filigran Community on Slack](https://community.filigran.io) 
     or submit your pull request on the [Github doc repository](https://github.com/OpenCTI-Platform/docs).
+# Export
+
+## Introduction
+
+With the OpenCTI platform, you can manually export your intelligence content in two formats:
+
+- structured with exports in .json and/or .csv formats,
```

Comment:
I wouldn't use "." before each extension and I would capitalize the first letter